### DISCUSSION TITLE
[Flags] Standardize names across CLI

### DISCUSSION THREAD

**Discussion by roethke** [ChainSafe/chainbridge-core#156] [CLI]
A discussion is started about standardizing flag names (perhaps also descriptions) across CLI commands.

These are some of the variants I’ve found across our CLIs commands:
- `target`
- `minter`
- `erc20Address`
- `token`
- `bridge`
- `tokenContract`
- `handler`

**The crux of the issue:** Since the ChainBridge CLI utilizes so many different flags with varying formats for its various commands, it is possible that a user could get confused; for example, `target`, versus `tokenContract`, versus `token`, versus `erc20Address`. In other words, why are we using **4** different formats for describing a contract address, whether that be a handler, an erc20 or the bridge contract itself? Additionally, using so many different formats looks unprofessional.

If we use one naming convention for a contract address, then that same naming convention should be applied to all flag names that are also contract addresses. Additionally, it is likely that much confusion for users could also be mitigated by better flag descriptions.

[Related Issue](https://github.com/ChainSafe/chainbridge-core/issues/105)

**Comment by P1sar** [comment]
I was already pointed that once, that variants are really depend on the context, because, for example, on AddMinter command, argument should be minter, on RegisterHandler it should be handlerAddress, but in both cases those will be erc20Handler addresses.
I do not like that we are discussing too abstract topic i'd rather consider specific CMD's
